dreammall-earth/dreammall.earth
# ci(workflow): speed up e2e tests

Motivation
----------
See this review: https://github.com/dreammall-earth/dreammall.earth/pull/1180#discussion_r1648992425

I want to challenge why we have a `sleep` in our workflow after all. I don't think it's necessary. I will let this run a couple of times and see where it breaks.

How to test
-----------
1. Restart the e2e workflow a number of times
2. See if it fails
